tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -343,6 +343,12 @@ TEST_F(FileTest, ByteAddressBufferStore) {
   runFileTest("method.byte-address-buffer.store.hlsl");
 }
 
+// For Buffer/RWBuffer methods
+TEST_F(FileTest, BufferLoad) { runFileTest("buffer.load.hlsl"); }
+TEST_F(FileTest, BufferOperatorBrackets) {
```

Comment:
This is named as "access" previously and put around Line 200. I'd suggest to rename it as `op.buffer.access.hlsl` to keep consistent with others.